### ISSUE TITLE
feature request: delay sending e-mails

### ISSUE THREAD

**Issue by cptMikky** [RainLoop/rainloop-webmail#779]
Mistakes happen. Wrong recipient, missing attachments... People realize their mistakes in the matter of seconds. But by then, the mail is usually long gone. Many e-mail clients, including webGmail have an option to delay sending e-mails. It would be very useful if RainLoop had a similar functionality - an option to delay e-mail sending.

**Comment by ervee** [comment]
I don't think this is possible without a Server backend process because Rainloop is (just) an IMAP/SMTP frontend. Who will send the delayed message when you close your browser or shutdown your computer?

**Comment by cptMikky** [comment]
Not neccessarily - a delay of 30-60 seconds is more than enought to catch most mistakes and this can be implemented in javascript. A "send now" button can be present for time-critical situations.

**Comment by arturbonnett** [comment]
Ideas on the tops of the images (buttons)...

On sending:
![on-sending](https://cloud.githubusercontent.com/assets/7110872/8960815/2d05442a-35e8-11e5-9545-11a3cbbb0b02.png)

After allegedly sent:
![after-allegedly-sent](https://cloud.githubusercontent.com/assets/7110872/8960828/384cfa94-35e8-11e5-9513-d1fa31197399.gif)

**Comment by farson2003** [comment]
I believe it is a valid idea, but if I was a developer I would not give that my top priority. Stability is more important than new features that help only few...

**Comment by ChebMignon** [comment]
I'm also very interested in this feature, its one of the thing that makes webmail Gmail really great. I believe a lot of people use it.

**Comment by AnisKhanUK** [comment]
Love Rainloop and it would be great to have a 30 second delay feature.  It has saved my bacon lots of times (it is built into OWA and gmail).  In fact, where I work, they have set it as default for everyone.

**Comment by CodeAlDente** [comment]
I think it is a completely useless functionality as it would emphasize us in a strange way to actually do stupid mistakes. Why? Because we can, as of the possibility to cancel the message to be sent. Today 30 seconds are enough just because we do not have this option right now. But later on we will end up with minutes to hours as it becomes more comfy to us.
